app/src/main/java/com/mupper/gobus/data/database/bus/BusDao.kt

```diff
@@ -15,7 +10,7 @@ import com.mupper.gobus.data.database.relations.BusWithTravelers
 @Dao
 interface BusDao {
     @Query("SELECT count(*) FROM bus WHERE path = :path")
-    fun getCount(path: String): Int
+    fun busCount(path: String): Int
```

Comment:
I think that, as a good practice, method names are typically verbs.

Reply:
Good catch!